core-services/prow/02_config/operator-framework/operator-controller/_pluginconfig.yaml

```
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: https://go.k8s.io/bot-commands
+  repos:
+  - operator-framework/operator-controller
+  require_self_approval: false
```

```suggestion
  require_self_approval: true
```

Is this the setting that controls what happens when approvers submit PRs? I personally feel like approvers that submit PRs should get a different approver to approve.

---

Certainly open for discussion.  This is 601 other prow-icized repos use `false` which prevails over `true` (182 instances), so I presumed it as SOP.

---

The logic for this is confusing (https://pkg.go.dev/sigs.k8s.io/prow/pkg/plugins). 
I believe that setting it `true` would require an approver to explicitly `/approve` a PR, whereas `false` would immediately `/approve` any approver-authored PR.